hi-ml-azure/run_requirements.txt

```diff
@@ -1,4 +1,5 @@
 azureml-sdk==1.32.0
+azure-storage-blob==12.9.0
```

Comment:
do we really need that in the run requirements, or only in the test_requirements?